### ISSUE TITLE
How many images do you use for validation?

### ISSUE THREAD

**Issue by waxnkw** [vincentschen/limited-label-scene-graphs#2]
I know that some papers use 5,000 images in the training set as the validation set. In 20 class settings, there are only around 10,000 images which contain  20 relationships used for classification.  How many images do you use for validation?

**Comment by waxnkw** [comment]
I found the download link of `split.npy` in `scripts/get_visualgenome.sh`.
